dapps/templates/boilerplate/package.json

```diff
@@ -33,20 +33,21 @@
   },
   "devDependencies": {
     "embark": "^5.3.0-nightly.0",
-    "embark-geth": "^5.3.0-nightly.0",
-    "embark-graph": "^5.3.0-nightly.0",
-    "embark-ipfs": "^5.3.0-nightly.0",
-    "embark-parity": "^5.3.0-nightly.0",
-    "embark-profiler": "^5.3.0-nightly.0",
     "embark-reset": "^5.2.3",
-    "embark-swarm": "^5.3.0-nightly.0",
-    "embark-whisper-geth": "^5.3.0-nightly.0",
     "embarkjs": "^5.2.3",
     "embarkjs-ens": "^5.2.3",
     "embarkjs-ipfs": "^5.2.3",
     "embarkjs-swarm": "^5.2.3",
     "embarkjs-web3": "^5.2.3",
     "embarkjs-whisper": "^5.2.3",
+    "embark-ipfs": "^5.3.0-nightly.0",
+    "embark-swarm": "^5.3.0-nightly.0",
+    "embark-whisper-geth": "^5.3.0-nightly.0",
+    "embark-geth": "^5.3.0-nightly.0",
+    "embark-parity": "^5.3.0-nightly.0",
+    "embark-profiler": "^5.3.0-nightly.0",
+    "embark-graph": "^5.3.0-nightly.0",
+    "embark-basic-pipeline": "^5.3.0-nightly.0",
```

Comment:
Why move them after? I think alphabetically, they should go before (that's how NPM puts them)

Reply:
I agree, it's better to maintain alpha sort. In fact, some tools will alpha sort and re-write `package.json` `"dependencies"` and `"devDependencies"`. So one day someone working on it will wonder why they got a surprise change-on-disk; better to keep it sorted in the first place.